libfreerdp/crypto/test/TestFuzzCryptoGetPublicKey.c

```diff
@@ -0,0 +1,8 @@
+#include <freerdp/crypto/crypto.h>
+
+int LLVMFuzzerTestOneInput(const uint8_t* Data, size_t Size)
+{
+	CryptoCert cert = malloc(sizeof(*cert));
+	crypto_cert_get_public_key(cert, (BYTE**)&Data, (DWORD*)Size);
```

Comment:
ok, you mixed up `input` and `output` variables, I think you want to check if it handles random input...